services/wallet/api.go

```diff
@@ -376,26 +397,31 @@ func (api *API) GetEthereumChains(ctx context.Context) ([]*network.CombinedNetwo
 	return api.s.rpcClient.NetworkManager.GetCombinedNetworks()
 }
 
+// @deprecated
 func (api *API) FetchPrices(ctx context.Context, symbols []string, currencies []string) (map[string]map[string]float64, error) {
 	log.Debug("call to FetchPrices")
 	return api.s.marketManager.FetchPrices(symbols, currencies)
 }
 
+// @deprecated
 func (api *API) FetchMarketValues(ctx context.Context, symbols []string, currency string) (map[string]thirdparty.TokenMarketValues, error) {
 	log.Debug("call to FetchMarketValues")
 	return api.s.marketManager.FetchTokenMarketValues(symbols, currency)
 }
 
+// @deprecated
 func (api *API) GetHourlyMarketValues(ctx context.Context, symbol string, currency string, limit int, aggregate int) ([]thirdparty.HistoricalPrice, error) {
```

Comment:
Used in building the asset historical value graph.

Reply:
Indeed